namespace java thrift.test

struct mySimpleStruct {
  1: bool a,
  2: byte b,
  3: required i16 c,
  4: i32 d,
  5: i64 e,
  6: double f,
  7: string g,
}
